packages/rancher-monitoring/generated-changes/patch/templates/_helpers.tpl.patch

```diff
@@ -73,6 +91,20 @@
 +{{- end -}}
 +{{- end }}
 +
++{{- define "exporter.kubelet.jobName" -}}
++{{- if .Values.k3sServer.enabled -}}
++k3s-server
++{{- else -}}
++kubelet
++{{- end -}}
++{{- end }}
++
++{{- define "exporter.controlplane.upLabels" -}}
++{{- if .Values.k3sServer.enabled -}}
++, metrics_path=\"/metrics\"
++{{- end -}}
++{{- end }}
++
```

Comment:
I see why you added this, but I think this should not be a helper function. We should patch the expression directly for this, since it only happens a couple of times.